src/Core/src/Core/IPlatformApplication.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Microsoft.Maui
```

Comment:
Just thinking if we should have this in this namespace? Or maybe it should be in .Platform? Just thinking aloud.